include/linux/soundwire/sdw_registers.h

```diff
@@ -114,12 +120,111 @@
 #define SDW_SCP_DEVID_4				0x54
 #define SDW_SCP_DEVID_5				0x55
 
+/* Both INT and STATUS register are same */
+#define SDW_SCP_SDCA_INT1			0x58
```

Comment:
@plbossart  Do we need to add SCP_SDCA_IntClear|IntStatX ?

Reply:
no, that's the point. you clear by writing to the status register, the names are different but the address is the same.